test/fixture/tty/color-disabled.js

```diff
@@ -1,6 +1,9 @@
 import test from '../../..';
 
-test('stdout does not support color', t => {
-	t.true(process.stdout.isTTY);
-	t.is(process.stdout.getColorDepth(), 1);
-});
+const assertNoColor = stream => t => {
+	t.true(stream.isTTY);
+	t.is(stream.getColorDepth(), 1);
+};
+
+test('stderr does not support color', assertNoColor(process.stderr));
```

Comment:
In all of these, you could write the assertion method like this:

```js
const assertNoColor = (t, stream) => {
  // assertions here
}

test('title', assertNoColor, process.stderr)
```

This uses [test macros](https://github.com/avajs/ava#test-macros).

Reply:
Oh, very cool. I hadn't found that feature yet—thanks for the tip!